src/get_estimation.py

```diff
@@ -2,32 +2,27 @@
 # -*- coding:utf-8 -*-
 
 
+import warnings
 import time
-from .benchmark_mediation import *
-from rpy2.rinterface_lib.embedded import RRuntimeError
 import sys
+from rpy2.rinterface_lib.embedded import RRuntimeError
 import pandas as pd
 import numpy as np
+from .benchmark_mediation import *
 
 
 def get_estimation(x, t, m, y, estimator, config):
+    effects = [np.nan] * 5
```

Comment:
Does this mean that the function can return np.nan ?
I don't think that this is a good idea. 
Initializing to zero would be more robust , no ?

Reply:
Yes it can return np.nan :
- if the estimator is not appropriated to the data (whether for theoretical or implementation reasons)
- if you misspell the estimator name

`benchmark_mediation.py` also returns some np.nan, in the multiply_robust estimator for instance (where half of the effects are not implemented yet).

I don't know either, @judithabk6 initialized it.
If we choose zero initialization, then we must review `benchmark_mediation.py` too.

Reply:
I think that if the estimator is not suited to the problem or is misspelled, the code should spit an error, period.

Reply:
I adressed the not suited estimator issue in #22, which a8fa379 should solve

Reply:
I will modify PR #20 accordingly, so that it can test if ValueError is returned